src/hidra/receiver/datareceiver.py

```diff
@@ -406,20 +497,13 @@ def _setup_logging(self):
     def _load_plugin(self):
         try:
             plugin_name = self.config["datareceiver"]["plugin"]
-            self.plugin_config = self.config[plugin_name]
+            plugin_config = self.config[plugin_name]
```

Comment:
this took me a while to understand -- the plugin becomes a command line parameter?

Reply:
The plugin can be configured in the yaml config file for the receiver.